### ISSUE TITLE
Обязательность payload в action аргументе sendAction

### ISSUE THREAD

**Issue by Skamnoel** [sberdevices/assistant-client#149]
Почему в sendAction для передаваемого action обязательный payload ?
https://github.com/sberdevices/assistant-client/blob/main/src/index.ts#L257

Кажется, что он должен быть необязательным, ведь под капотом sendData, который в итоге использует:
https://github.com/sberdevices/assistant-client/blob/main/src/typings.ts#L94

**Comment by sasha-tlt** [comment]
Да, кажется что необязательно передавать `payload`. Сделаем вместе с #130
